python/README.md

```diff
@@ -0,0 +1,56 @@
+# This folder contains manual bindings for aikidopy using pybind11.
```

Comment:
Nit: this header shouldn't be a sentence. Maybe something like
```suggestion
# aikidopy
```
with that sentence as just regular text would make more sense.